FightPandemics/Views/Components/Cards/Feed.swift

```diff
@@ -0,0 +1,117 @@
+    }
+    
+    var metaInfo: some View {
+        HStack(alignment: .top) {
+            HStack(spacing: 10) {
+                Text("Requests")
+                Ellipse()
+                    .foregroundColor(Color("purpleishBlue"))
+                    .frame(width: 2, height: 2)
+                Text("Posted xhrs ago")
+                    .foregroundColor(Color("brownGrey"))
+            }
+            .foregroundColor(Color("purpleishBlue"))
+            .font(.iosNativeP311Px)
+            Spacer()
+            TagSelected()
+        }
+    }
+    
+    var userInfo: some View {
+        HStack(spacing: 10) {
+            ZStack {
+                Circle()
+                    .strokeBorder(Color("purpleishBlue"), lineWidth: 1)
+                    .background(Circle().fill(Color("paleGreyTwo")))
+                    .frame(width: 36, height: 36)
+                Text("NN")
+                    .font(.iosNativeH314Px)
+                    .foregroundColor(Color("purpleishBlue"))
+            }
+            VStack(alignment: .leading, spacing: 0) {
+                Text("User Name")
+                    .font(.iosNativeH314Px)
+                    .lineSpacing(18)
+                HStack(alignment: .center, spacing: 4) {
+                    Ellipse()
+                        .foregroundColor(Color("purpleishBlue"))
+                        .frame(width: 6, height: 6)
+                    Text("City, Country")
+                        .font(.iosNativeP311Px)
+                        .foregroundColor(Color("brownGrey"))
+                        .lineSpacing(15.4)
+                }
+            }
+        }
+    }
+    
+    var socialButtons: some View {
+        HStack() {
+            HStack(spacing: 8) {
+                Button(action: {
+                    print("Like Button Tapped!")
+                }) {
+                    Image("heart-filled-icon")
+                        .frame(width: 24, height: 24)
+                }
+                Text("12")
+                    .font(.iosNativeP212Px)
+                    .foregroundColor(Color("brownGreyTwo"))
+            }
+            Spacer().frame(width: 28)
+            HStack(spacing: 8) {
+                Image("msg-icon")
+                    .frame(width: 24, height: 24)
+                Text("3")
+                    .font(.iosNativeP212Px)
+                    .foregroundColor(Color("brownGreyTwo"))
+            }
+            Spacer().frame(width: 31)
+            Button(action: {
+                print("Share Button Tapped!")
+            }) {
+                Image("share-icon")
+                    .frame(width: 24, height: 24)
+            }
+        }
+    }
+}
+
+struct Feed_Previews: PreviewProvider {
+    static var previews: some View {
+        Group {
+            Feed()
+                .previewLayout(.fixed(width: 395, height: 362))
+                .environment(\.colorScheme, .light)
+        }
+        
+    }
+}
```

Comment:
Good!
Maybe you can extract this UI pieces on differents structs-files (modularization)
Example: MetaInfoView, SocialButtonsView, etc